source/api/presentation/3.0/index.md

```diff
@@ -1513,7 +1540,7 @@ Recipes for enabling CORS, conditional Content-Type headers and other technical
 
 ## 7. Authentication
 
-It may be necessary to restrict access to the descriptions made available via the Presentation API.  As the primary means of interaction with the descriptions is by web browsers using XmlHttpRequests across domains, there are some considerations regarding the most appropriate methods for authenticating users and authorizing their access.  The approach taken is described in the [Authentication][auth] specification, and requires requesting a token to add to the requests to identify the user.  This token might also be used for other requests defined by other APIs.
+It may be necessary to restrict access to the descriptions made available via the Presentation API.  As the primary means of interaction with the descriptions is by web browsers using XmlHttpRequests across domains, there are some considerations regarding the most appropriate methods for authenticating users and authorizing their access.  The approach taken is described in the [Authentication][auth-api] specification, and requires requesting a token to add to the requests to identify the user.  This token might also be used for other requests defined by other APIs.
```

Comment:
Style question: should XmlHttpRequests be `XmlHttpRequest`s?

Reply:
Maybe? We don't tend to `backtick` our own class names, so I'd be at least hesitant to do it here ... but I don't think we have any consistency.

Reply:
Make a separate ticket and move on? This is easily solved in a pub.

Reply:
Yup, 👍  Will make the issue now.

Reply:
--> #1371